lib/dictionary

```diff
@@ -0,0 +1,168 @@
+personal_ws-1.1 en 0
+actionables
+AdVenture
+AIME
+AMC
+api
+API
+APIs
+Arduino
+bannings
+Bloomberg
+bogidon
+bolded
+cancelled
+CAPTCHA
+checkbox
+checkboxes
+Christabel
+Chromebook
+Chromebooks
+Codecademy
+CodeHS
+could've
+cringiest
+cringy
+CSS
+CTF
+CTFs
+Ctrl
+CVV
+DDoS
+de
+decrement
+Django
+Dropbox
+EasyCTF
+EasyCTF's
+Flappy
+flyer
+flyers
+FreeBSD
+freenode
+Fu
+fundraise
+Gaberiel
+GIF
+GIFs
+girlsWhoCode
```

Comment:
Ah, cool. Just submitted #52 and hackclub/hackclub#632 to fix this :-).

On Sun, Jan 3, 2016 at 2:14 PM, Samuel Escapa <notifications@github.com>
wrote:

> In lib/dictionary
> <https://github.com/hackclub/markcop/pull/50#discussion_r48697919>:
>
> > +decrement
> > +Django
> > +Dropbox
> > +EasyCTF
> > +EasyCTF's
> > +Flappy
> > +flyer
> > +flyers
> > +FreeBSD
> > +freenode
> > +Fu
> > +fundraise
> > +Gaberiel
> > +GIF
> > +GIFs
> > +girlsWhoCode
>
> I just looked it up, it's Girls Who Code.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/markcop/pull/50/files#r48697919>.
>



-- 
Zach Latta
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta